libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1400,6 +1403,11 @@ - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator displayConfirmationMe
     [[SFRootViewManager sharedManager] pushViewController:alert];
 }
 
+- (void)oauthCoordinatorDidCancelBrowserAuthentication:(SFOAuthCoordinator*)coordinator
+{
+    [self delegateDidCancelBrowserFlow];
```

Comment:
I'm not seeing in the flow where we consider authentication over if canceled.  Where does that cancelation occur?